.yamato/_abv.yml

```diff
@@ -7,51 +7,52 @@
+    name: ABV (all PR jobs) on 2019.4
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
     dependencies:
-      -  path: .yamato/all-universal.yml#PR_Universal_2019.4
-         rerun: on_new_revision
-      -  path: .yamato/all-shadergraph.yml#PR_ShaderGraph_2019.4
-         rerun: on_new_revision
       -  path: .yamato/all-hdrp.yml#PR_HDRP_2019.4
          rerun: on_new_revision
-      -  path: .yamato/all-vfx_hdrp.yml#PR_VFX_HDRP_2019.4
+      -  path: .yamato/all-vfx.yml#PR_VFX_2019.4
          rerun: on_new_revision
-      -  path: .yamato/all-vfx_urp.yml#PR_VFX_URP_2019.4
-         rerun: on_new_revision
-      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2019.4
-         rerun: on_new_revision
-all_project_ci_weekly_2019.4:
-    name: _Weekly ABV against 2019.4
-    dependencies:
-      -  path: .yamato/all-universal.yml#PR_Universal_2019.4
+      -  path: .yamato/all-urp.yml#PR_URP_2019.4
          rerun: on_new_revision
       -  path: .yamato/all-shadergraph.yml#PR_ShaderGraph_2019.4
          rerun: on_new_revision
-      -  path: .yamato/all-hdrp.yml#PR_HDRP_2019.4
-         rerun: on_new_revision
-      -  path: .yamato/all-vfx_hdrp.yml#PR_VFX_HDRP_2019.4
+      -  path: .yamato/_abv.yml#General_Checks_2019.4
          rerun: on_new_revision
-      -  path: .yamato/all-vfx_urp.yml#PR_VFX_URP_2019.4
+all_project_ci_weekly_2019.4:
+    name: Weekly on 2019.4
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+    dependencies:
+      -  path: .yamato/all-hdrp.yml#Nightly_HDRP_2019.4
          rerun: on_new_revision
-      -  path: .yamato/smoke_test-win-dx11.yml#Smoke_Test_Win_DX11_playmode_mono_Linear_2019.4
+      -  path: .yamato/all-vfx.yml#Nightly_VFX_2019.4
          rerun: on_new_revision
-      -  path: .yamato/smoke_test-win-dx11.yml#Smoke_Test_Win_DX11_editmode_mono_Linear_2019.4
+      -  path: .yamato/all-urp.yml#Nightly_URP_2019.4
          rerun: on_new_revision
-      -  path: .yamato/smoke_test-win-dx11.yml#Smoke_Test_Win_DX11_Standalone_mono_Linear_2019.4
+      -  path: .yamato/all-shadergraph.yml#Nightly_ShaderGraph_2019.4
          rerun: on_new_revision
-      -  path: .yamato/universal-android-opengles3.yml#Universal_Android_OpenGLES3_Standalone_il2cpp_Linear_2019.4
+      -  path: .yamato/_abv.yml#General_Checks_2019.4
          rerun: on_new_revision
-      -  path: .yamato/universal-android-vulkan.yml#Universal_Android_Vulkan_Standalone_il2cpp_Linear_2019.4
+General_Checks_2019.4:
+    name: General Checks on 2019.4
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+    dependencies:
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2019.4
          rerun: on_new_revision
-      -  path: .yamato/universal-iphone-metal.yml#Universal_iPhone_Metal_Standalone_il2cpp_Linear_2019.4
+      -  path: .yamato/all-smoke_test.yml#Smoke_Test_2019.4
          rerun: on_new_revision
-    triggers:
```

Comment:
I think the weekly trigger was accidentally removed?

Reply:
Good catch this one too, will push a fix!